proto/coprocessor.proto

```diff
@@ -9,22 +9,24 @@ option (gogoproto.marshaler_all) = true;
 option (gogoproto.sizer_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
+option java_package = "com.pingcap.tikv.grpc";
```

Comment:
I'm fine for rename. Previously all coprocessor related are under tikvproto with prefix grpc.